salt/apps/ocw/nextgen_build_publish.sls

```diff
@@ -0,0 +1,31 @@
+{% set ocw_next = salt.pillar.get('ocw-next') %}
+
+run_ocw_to_hugo:
+  cmd.run:
+    # Installing ocw-to-hugo without the `-g' switch leaves us without an
+    # `ocw-to-hugo' executable, so we use the path to `index.js'
+    - name: node src/bin/index.js -i /home/ocw/open-learning-course-data -o /home/ocw/hugo-course-publisher/site/content --strips3 --staticPrefix /coursemedia
+    # `cwd' is specified because it drops a log file here.
+    - cwd: /home/ocw/ocw-to-hugo
+    - runas: ocw
+
+run_hugo_course_publisher:
+  cmd.run:
+    - name: npm run build
+    # `cwd' is specified because it drops a log file here.
+    - cwd: /home/ocw/hugo-course-publisher
+    - runas: ocw
+    - env:
+      - NODE_OPTIONS: '-max_old_space_size=8192'
```

Comment:
We don't need this value. That was a relic of an effort to resolve a bug that was due to duplicated content and the Hugo process, not the Node process.

Reply:
Removed in 0711b181